packages/grafana-data/src/field/overrides/processors.ts

```diff
@@ -12,12 +12,21 @@ export const identityOverrideProcessor = <T>(value: T, _context: FieldOverrideCo
   return value;
 };
 
+export interface Bound {
+  path: string;
+  offest: number;
+  defaultBound: number;
+  dynamic?: boolean;
+}
```

Comment:
Please add docs comments for this interface - will be easier to reason about it in the future for people who are not famiiar with this code.